diff --git a/node_modules/react-native-hyperlink/dist/index.js b/node_modules/react-native-hyperlink/dist/index.js
index 4caa87a..a771fa4 100644
--- a/node_modules/react-native-hyperlink/dist/index.js
+++ b/node_modules/react-native-hyperlink/dist/index.js
@@ -47,7 +47,7 @@ function _defineProperty(obj, key, value) { if (key in obj) { Object.definePrope
 
 var linkify = require('linkify-it')();
 
-var textPropTypes = _reactNative.Text.propTypes || {};
+var textPropTypes = require('deprecated-react-native-prop-types').TextPropTypes || {};
 var OS = _reactNative.Platform.OS;
 
 var Hyperlink =
diff --git a/node_modules/react-native-hyperlink/src/Hyperlink.js b/node_modules/react-native-hyperlink/src/Hyperlink.js
index 627c78d..75d4f55 100644
--- a/node_modules/react-native-hyperlink/src/Hyperlink.js
+++ b/node_modules/react-native-hyperlink/src/Hyperlink.js
@@ -10,10 +10,11 @@ import {
 	Platform
 } from 'react-native'
 import mdurl from 'mdurl'
+import {TextPropTypes} from 'deprecated-react-native-prop-types';
 
 const linkify = require('linkify-it')()
 
-const textPropTypes = Text.propTypes || {}
+const textPropTypes = TextPropTypes || {}
 const { OS } = Platform
 
 class Hyperlink extends Component {
