diff --git a/node_modules/react-native-image-gallery/src/Gallery.js b/node_modules/react-native-image-gallery/src/Gallery.js
index 5cc4835..8801c6a 100644
--- a/node_modules/react-native-image-gallery/src/Gallery.js
+++ b/node_modules/react-native-image-gallery/src/Gallery.js
@@ -1,9 +1,10 @@
 import React, { PureComponent } from 'react';
-import { View, ViewPropTypes } from 'react-native';
+import { View } from 'react-native';
 import PropTypes from 'prop-types';
 import { createResponder } from './libraries/GestureResponder';
 import TransformableImage from './libraries/TransformableImage';
 import ViewPager from './libraries/ViewPager';
+import {ViewPropTypes} from 'deprecated-react-native-prop-types';
 
 const DEFAULT_FLAT_LIST_PROPS = {
     windowSize: 3
diff --git a/node_modules/react-native-image-gallery/src/libraries/TransformableImage/index.js b/node_modules/react-native-image-gallery/src/libraries/TransformableImage/index.js
index 8ff8469..992c73b 100644
--- a/node_modules/react-native-image-gallery/src/libraries/TransformableImage/index.js
+++ b/node_modules/react-native-image-gallery/src/libraries/TransformableImage/index.js
@@ -1,5 +1,6 @@
 import React, { PureComponent } from 'react';
-import { View, Text, Image, ViewPropTypes } from 'react-native';
+import { View, Text, Image} from 'react-native';
+import {ViewPropTypes} from 'deprecated-react-native-prop-types';
 import PropTypes from 'prop-types';
 import ViewTransformer from '../ViewTransformer';
 
diff --git a/node_modules/react-native-image-gallery/src/libraries/ViewPager/index.js b/node_modules/react-native-image-gallery/src/libraries/ViewPager/index.js
index b786ad1..3a1a558 100644
--- a/node_modules/react-native-image-gallery/src/libraries/ViewPager/index.js
+++ b/node_modules/react-native-image-gallery/src/libraries/ViewPager/index.js
@@ -2,10 +2,10 @@ import React, { PureComponent } from 'react';
 import {
     View,
     FlatList,
-    ViewPropTypes,
     InteractionManager,
     Dimensions
 } from 'react-native';
+import {ViewPropTypes} from 'deprecated-react-native-prop-types';
 import PropTypes from 'prop-types';
 import Scroller from '../Scroller';
 import { createResponder } from '../GestureResponder';
diff --git a/node_modules/react-native-image-gallery/src/libraries/ViewTransformer/index.js b/node_modules/react-native-image-gallery/src/libraries/ViewTransformer/index.js
index f919e99..9e6c48b 100644
--- a/node_modules/react-native-image-gallery/src/libraries/ViewTransformer/index.js
+++ b/node_modules/react-native-image-gallery/src/libraries/ViewTransformer/index.js
@@ -1,5 +1,5 @@
 import React from 'react';
-import ReactNative, { View, Animated, Easing, NativeModules } from 'react-native';
+import ReactNative, { View, Animated, Easing, UIManager } from 'react-native';
 import Scroller from '../Scroller';
 import PropTypes from 'prop-types';
 import { createResponder } from '../GestureResponder';
@@ -162,7 +162,7 @@ export default class ViewTransformer extends React.Component {
 
     measureLayout () {
         let handle = ReactNative.findNodeHandle(this.refs['innerViewRef']);
-        NativeModules.UIManager.measure(handle, (x, y, width, height, pageX, pageY) => {
+        UIManager.measure(handle, (x, y, width, height, pageX, pageY) => {
             if (typeof pageX === 'number' && typeof pageY === 'number') { // avoid undefined values on Android devices
                 if (this.state.pageX !== pageX || this.state.pageY !== pageY) {
                     this.setState({ pageX: pageX, pageY: pageY });
