diff --git a/node_modules/react-native-screens/lib/commonjs/components/Screen.js b/node_modules/react-native-screens/lib/commonjs/components/Screen.js
index e3887b8..84dc0f5 100644
--- a/node_modules/react-native-screens/lib/commonjs/components/Screen.js
+++ b/node_modules/react-native-screens/lib/commonjs/components/Screen.js
@@ -88,7 +88,6 @@ const InnerScreen = exports.InnerScreen = /*#__PURE__*/_react.default.forwardRef
       ...props
     } = rest;
     if (active !== undefined && activityState === undefined) {
-      console.warn('It appears that you are using old version of react-navigation library. Please update @react-navigation/bottom-tabs, @react-navigation/stack and @react-navigation/drawer to version 5.10.0 or above to take full advantage of new functionality added to react-native-screens');
       activityState = active !== 0 ? 2 : 0; // in the new version, we need one of the screens to have value of 2 after the transition
     }
     if (isNativeStack && prevActivityState !== undefined && activityState !== undefined) {
diff --git a/node_modules/react-native-screens/lib/module/components/Screen.js b/node_modules/react-native-screens/lib/module/components/Screen.js
index f6a666f..a95c483 100644
--- a/node_modules/react-native-screens/lib/module/components/Screen.js
+++ b/node_modules/react-native-screens/lib/module/components/Screen.js
@@ -84,7 +84,6 @@ export const InnerScreen = /*#__PURE__*/React.forwardRef(function InnerScreen(pr
       ...props
     } = rest;
     if (active !== undefined && activityState === undefined) {
-      console.warn('It appears that you are using old version of react-navigation library. Please update @react-navigation/bottom-tabs, @react-navigation/stack and @react-navigation/drawer to version 5.10.0 or above to take full advantage of new functionality added to react-native-screens');
       activityState = active !== 0 ? 2 : 0; // in the new version, we need one of the screens to have value of 2 after the transition
     }
     if (isNativeStack && prevActivityState !== undefined && activityState !== undefined) {
diff --git a/node_modules/react-native-screens/src/components/Screen.tsx b/node_modules/react-native-screens/src/components/Screen.tsx
index d23d6f3..48a741d 100644
--- a/node_modules/react-native-screens/src/components/Screen.tsx
+++ b/node_modules/react-native-screens/src/components/Screen.tsx
@@ -145,9 +145,6 @@ export const InnerScreen = React.forwardRef<View, ScreenProps>(
       } = rest;
 
       if (active !== undefined && activityState === undefined) {
-        console.warn(
-          'It appears that you are using old version of react-navigation library. Please update @react-navigation/bottom-tabs, @react-navigation/stack and @react-navigation/drawer to version 5.10.0 or above to take full advantage of new functionality added to react-native-screens',
-        );
         activityState = active !== 0 ? 2 : 0; // in the new version, we need one of the screens to have value of 2 after the transition
       }
 
